### PR TITLE
Release 08 10 15

### PR DESCRIPTION
**Pull request by bmrsny**
@threatsim/development Please Review!

### CODE REVIEW

**Comment by bhargavisatpathy** [comment]
:+1:

**Comment by bmrsny** [comment]
When using rake preview, my avatar shows as Nina??? I believe I changed all the correct info for the post. confused.

**Comment by mastastealth** [comment]
Stealing identities is a crime, @bmrsny

**Comment by veverkap** [comment]
That's my mistake @bmrsny - I copy pasted badly.  I'll fix.

File: _posts/2015-08-07-I-am-new-here.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Initially, I considered going back to the local University and majoring in Computer Science. While preparing to go back to school, a fellow co-worker mentioned a few intensive coding schools around town. I researched two in particular. After attending a demo at [Turing School of Software and Design](http://turing.io/) and speaking with a few students and the executive director, I knew this was the place. I applied almost immediately. A few days later I received an acceptance letter. My gut had led me this far, so I went all in.
+
+Turing school was the most strenuous challenge I have ever attempted. After 7 months of intensive software development training, I was released into the wild world of the job hunt. Shortly after, ThreatSim hired three talented female developers and myself, all of us Turing School graduates. At first, I wasn't sold on the idea of working remotely. However, After I spoke with developers Patrick and Julian, I really hoped that they would ask me to join the team. Houston we have landed, I got the job!
```
Comment (veverkap) [inline _posts/2015-08-07-I-am-new-here.md]: Should you capitalize the s in school?

Comment (veverkap) [inline _posts/2015-08-07-I-am-new-here.md]: Also:

```
However, after I spoke with ...
```

**Comment by bmrsny** [comment]
Making more changes! I will update when I am finished.

**Comment by bmrsny** [comment]
Ready to be reviewed again!

**Pull request merged**